cmds/vpdbootmanager/add.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/booter"
+	"github.com/systemboot/systemboot/pkg/vpd"
+)
+
+var dryRun = false
+
+func add(entrytype string, args []string) error {
+	var entry booter.Booter
+	var err error
+	switch entrytype {
+	case "netboot":
+		if len(args) < 2 {
+			return fmt.Errorf("You need to pass method and MAC address")
+		}
+		entry, err = parseNetbootFlags(args[0], args[1], args[2:])
+		if err != nil {
+			return err
+		}
+	case "localboot":
+		if len(args) < 1 {
+			return fmt.Errorf("You need to provide method")
+		}
+		entry, err = parseLocalbootFlags(args[0], args[1:])
+		if err != nil {
+			return err
+		}
+	default:
+		return fmt.Errorf("Unknown entry type")
+	}
+	if dryRun {
+		b, err := json.Marshal(entry)
+		if err != nil {
+			return err
+		}
+		fmt.Println("Using -dryrun, will not write any variable. Content of boot entry:")
```

Comment:
can you print this on stderr? So that the JSON output can be redirected if desired